lib/static/modules/reducers/static-image-accepter.js

```diff
@@ -0,0 +1,199 @@
+import {get, set, flatMap, last} from 'lodash';
+import actionNames from '../action-names';
+import {checkIsEnabled, getLocalStorageCommitedImageIds} from '../static-image-accepter';
+import {applyStateUpdate, isAcceptable} from '../utils';
+import {COMMITED, STAGED} from '../../../constants';
+
+export default (state, action) => {
```

Comment:
State transition table.

"*base -> commit" - image is commited without being staged. It happens if you will commit accepted in "screenshot accepter" images without exiting it.

c3 calculated once inside "initStaticImageAccepter"

<img width="978" alt="image" src="https://github.com/gemini-testing/html-reporter/assets/54893992/1910ecff-0d35-4429-84ca-2dc0f98202f2">